code/game/objects/items/weapons/storage/belt.dm

```diff
@@ -173,25 +173,28 @@
 		/obj/item/weapon/reagent_containers/food/drinks/bottle,
 		/obj/item/stack/medical,
 		/obj/item/weapon/kitchen/knife,
-		/obj/item/weapon/reagent_containers/hypospray/medipen,
+		/obj/item/weapon/reagent_containers/hypospray,
 		/obj/item/device/gps,
 		/obj/item/weapon/storage/bag/ore,
 		/obj/item/weapon/survivalcapsule,
 		/obj/item/device/t_scanner/adv_mining_scanner,
 		/obj/item/weapon/reagent_containers/pill,
-		/obj/item/weapon/ore/bluespace_crystal,
-		/obj/item/weapon/reagent_containers/food/drinks,
-
-
+		/obj/item/weapon/storage/pill_bottle,
+		/obj/item/weapon/ore,
+		/obj/item/weapon/reagent_containers/food/drinks
 		)
 
+/obj/item/weapon/storage/belt/mining/vendor

```

Comment:
This is already the most chosen item from the vendor, I don't think it needs a buff